sdk/testing/src/main/java/io/opentelemetry/sdk/testing/assertj/OpenTelemetryAssertions.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.testing.assertj;
+
+import io.opentelemetry.api.common.ReadableAttributes;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import org.assertj.core.api.Assertions;
+
+/**
+ * Entry point for assertion methods for OpenTelemetry types. To use type-specific assertions,
+ * static import any {@code assertThat} method in this class instead of {@code
+ * Assertions.assertThat}.
+ */
+public class OpenTelemetryAssertions extends Assertions {
```

Comment:
Private constructor?

Reply:
Made it protector - a user might extend this into their own custom assertions class, doesn't hurt since just a bag of statics.

Reply:
Extending would also work with a private ctor I think. You just can't instantiate the derived class. But protected is OK for me here.

Reply:
Yeah thought it might work but when I tried it, it didn't :)